wagtailnhsukfrontend/blocks.py

```diff
@@ -82,3 +82,19 @@ class PanelBlock(StructBlock):
 
     class Meta:
         template = 'wagtailnhsukfrontend/panel.html'
+
+class DoDontListItemBlock(RichTextBlock):
+
+    class Meta:
+        template = 'do_dont_list_item.html'
+
+class DoDontBlock(StructBlock):
+
+    type = ChoiceBlock([
+        ('do', 'Do List'),
+        ('dont', 'Dont List'),
+    ], required=True)
+    listitems = ListBlock(DoDontListItemBlock)
```

Comment:
Do do/dont lists always have one block of dos and one block of donts?

If so, you could have
```
do = ListBlock(RichTextBlock)
dont = ListBlock(RichTextBlock)
```

That would remove the need for selecting "do" or "dont" on each item.